src/resources/parser/texture/img.js

```diff
@@ -84,6 +89,20 @@ Object.assign(ImgParser.prototype, {
         };
 
         image.src = url;
+    },
+
+    _loadImageBitmap: function (url, originalUrl, crossOrigin, callback) {
```

Comment:
Do we need to duplicate the retry logic here too?

Reply:
Yes, possibly. I may just use http instead of promise and bring the download method in line with other image types.

Reply:
Yeah, sounds sensible.